pep/app/session-stores/application.js

```diff
@@ -1,10 +1,10 @@
 import CookieStore from 'ember-simple-auth/session-stores/cookie';
 import ENV from 'pep/config/environment';
 
-export const SESSION_COOKIE_NAME = 'pep_session';
+export const SESSION_COOKIE_NAME = 'ember_simple_auth-session';
 
 export default CookieStore.extend({
-    cookieName: SESSION_COOKIE_NAME,
-    cookieExpirationTime: 60 * 60 * 24 * 365, // 1 year
-    sameSite: ENV.cookieSameSite
+    // cookieName: SESSION_COOKIE_NAME,
+    // cookieExpirationTime: 60 * 60 * 24 * 365 // 1 year
```

Comment:
won't using the default expiration here cause the cookie to be deleted when the browser is closed?

https://ember-simple-auth.com/api/classes/CookieStore.html#property_cookieExpirationTime

Reply:
Yes. Was it a requirement for it to last a year? Just trying to keep this to the default if possible

Reply:
Thats what Alan had me set it to when I had originally asked him. I think the thought was that since we dont get the real session expiration back from the API upon login, we would just make it something super long, to ensure that its always the API that invalidates the user's session.

I feel like for normal user logins, they are gonna kick this back as a bug, if you get logged out whenever the browser is closed, but would check w/ @adistasio to be sure.